pom.xml

```diff
@@ -12,6 +12,7 @@
   <packaging>pom</packaging>
   <name>Quarkus - Quinoa - Parent</name>
    <description>Quinoa is a Quarkus extension which eases the development, the build and serving of single page apps or web components alongside Quarkus. It is possible to use it with a Quarkus backend in a single project.</description>
+    <url>https://github.com/quarkiverse/quarkus-quinoa</url>
```

Comment:
Looks like there is an extra space at the start it should line up with description?